nautobot/ipam/models.py

```diff
@@ -870,6 +890,8 @@ class Meta:
         verbose_name = "VLAN group"
         verbose_name_plural = "VLAN groups"
 
+    _natural_key = ["name", "location"]
```

Comment:
Not actually a uniqueness constraint on VLANGroup due to the good old NULL != NULL problem. But close for the moment.

Reply:
So what happens if `location` is null in this instance?

Reply:
if we have two VLANGroups with the same name and a null location, then get_by_natural_key will be unhappy:

```python
>>> VLANGroup.objects.first()
<VLANGroup: AGE>
>>> VLANGroup.objects.create(name="AGE")
<VLANGroup: AGE>
>>> VLANGroup.objects.get_by_natural_key("AGE")
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/source/nautobot/core/models/__init__.py", line 52, in get_by_natural_key
    return self.get(**kwargs)
  File "/usr/local/lib/python3.7/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/cacheops/query.py", line 351, in get
    return qs._no_monkey.get(qs, *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/django/db/models/query.py", line 442, in get
    num if not limit or num < limit else 'more than %s' % (limit - 1),
nautobot.ipam.models.VLANGroup.MultipleObjectsReturned: get() returned more than one VLANGroup -- it returned 2!
```